dao/src/main/resources/db/changelog/logs/20221002-ch-add-column-creator-id-in-tariffs-info-table-Melnyk.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="Melnyk-7" author="Bohdan Melnyk">
+
+        <addColumn tableName="tariffs_info">
```

Comment:
why are you keeping separately add column for tariffs_info? You've created this table in 20220902-ch-add-table-tariffs-info-Melnyk.xml so why don't you declare all the needed columns there?